spec/rrdcached_proxy/rrdtool_connection_spec.rb

```diff
@@ -24,8 +24,12 @@ def sent_data
   let(:em_signature) { 1337 }
   let(:backend) { double(RRDCachedProxy::Backends::Base) }
   let(:socket) { '/var/run/rrdcached.sock' }
+  let(:blacklist) { nil }
 
-  let(:instance) { TestConnection.new(em_signature, logger, backend, socket) }
+  let(:instance) { TestConnection.new(em_signature, logger: logger,
```

Comment:
Avoid using {...} for multi-line blocks.<br>Block body expression is on the same line as the block start.